### DISCUSSION TITLE
Colab Python Version Issue : Stellar graph version not support to the current python version on colab

### DISCUSSION THREAD

**Discussion by VenkateshwaranB** [stellargraph/stellargraph#2083] [General] [answered]
In google CoLab, the python current version is 3.10 something, but the stellar graph library only supports python version 3.6.0. I tried to change the version but I facing too many new issues with this. So please provide a solution for that. 

Please refer to the image.

![colab error](https://user-images.githubusercontent.com/48711041/228021705-3fb2fb9f-6a16-4d8a-bd4c-03545659735a.png)

Thanks 
Venkat

**Comment by 666eternalatake** [comment]
Hi, have you resolved this issue?

**Comment by AmberXie** [comment]
Hi, have you solved this? I have the same problem.

**Reply by VenkateshwaranB** [reply, marked as answer]
yes, I resolved.

check my repository.
https://github.com/VenkateshwaranB/stellargraph.git

Check this one:
https://github.com/stellargraph/stellargraph/issues/2084
